### PR TITLE
 [tests-only] [full-ci] adding tests for shareWithMe endpoint when resource is shared with a group

### PR DESCRIPTION
**Pull request by ghost**
## Description

Scenarios added :
```feature
Scenario: user lists a file shared with them in a group from personal space
Scenario: user lists the file shared with them in a group from project space
Scenario: user lists a folder shared with them in a group from personal space
Scenario: user lists the folder shared with them in a group from project space
```

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/7826

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 1]
```diff
@@ -2974,4 +2974,1304 @@ Feature: an user gets the resources shared to them
...
+                              {
+                                "type": "string",
+                                "pattern": "^%role_id_pattern%$"
+                              }
+                            ]
+                          }
+                        }
+                      },
+                      {
+                        "type": "object",
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:3255]: sharing to group returns two permissions for the user in that group?

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 2]
```diff
@@ -2974,4 +2974,970 @@ Feature: an user gets the resources shared to them
...
+                      "driveType" : {
+                        "type": "string",
+                        "enum": ["personal"]
+                      }
+                    }
+                  },
+                  "permissions": {
+                    "type": "array",
+                    "maxItems":1,
+                    "items": [
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:3170]: helps make it more strict
```suggestion
                    "maxItems": 1,
                    "minItems": 1,
                    "items": [
```

**Review by saw-jan — APPROVED** [review]
:+1:

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 3]
```diff
@@ -1951,3 +1951,973 @@ Feature: an user gets the resources shared to them
...
+              "size"
+            ],
+            "properties": {
+              "@UI.Hidden":{
+                "type": "boolean",
+                "enum": [false]
+              },
+              "@client.synchronize":{
+                "type": "boolean",
+                "enum": [true]
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:1999]:
```suggestion
                "enum": [false]
```

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Pull request merged**